app/javascript/components/ingredients/ingredients-form.vue

```diff
@@ -111,6 +144,7 @@
 </template>
 
 <script>
+import baseModal from '../base/base-modal.vue';
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'baseModal' is defined but never used                  no-unused-vars